hack/testing/README.md

```diff
@@ -38,7 +38,47 @@ to have been created (based on the logs of the Elasticsearch pods).
 This Go script will query the appropriate ES pod (if an OPS cluster was used)
 and check the local log files for the each result message.
 
-## check-curator.sh
+## Environment values for testing
+
+In addition to providing the argument `true` to `e2e-test.sh` to indicate that
+an OPS cluster is used, the following environment values can be used to
+change behavior of the scripts.
+
+The format for the following would be
+`ENV_VAR1=value [ENV_VAR2=value2] ./e2e-test.sh`
+
+```
+VERBOSE                 | If set, the scripts will print out each command as executed | default: "" (unset)
```

Comment:
You probably do not want to mix `set -o xtrace` with `os::cmd` -- the latter is designed to give you all the information you need. If you run with `set -o xtrace` you get to see the (amazingly verbose) gory detail of the `os::cmd` implementation and I'm not sure anyone needs that.